### ISSUE TITLE
non-static method ActionScheduler_AdminHelp::add_help_tabs() should not be called statically

### ISSUE THREAD

**Issue by alessandrotesoro** [Prospress/action-scheduler#334]
Hi there,

I just wanted to report that I'm getting the following error:

```
call_user_func_array() expects parameter 1 to be a valid callback, non-static method ActionScheduler_AdminHelp::add_help_tabs() should not be called statically
```

Thank you for this library :)

**Comment by rrennick** [comment]
@alessandrotesoro Thanks for reporting this. I've created #335 to fix the issue.

**Comment by thenbrent** [comment]
Fixed in `version_3_0_0` with SHA: dcf05de

For more background, see https://github.com/Prospress/action-scheduler/pull/335#issuecomment-514001428
